### ISSUE TITLE
[$250] You" Excluded When Selecting account Email in Share-somewhere Section

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#39764]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.60-6
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:**

## Action Performed:
Prerequisite: Please use a long display name (if the default display name is used, ensure the email address is long).
1, sign in with your Gmail account > Click on FAB and select Assign task.
2, Add a title and click next.
3, Go to share somewhere and select your account email (the email used to sign in).
4, Notice that "You" is excluded.

## Expected Result:
When selecting one's account email (the email used to sign in) in Share Somewhere, "You" should not be excluded.

## Actual Result:
When selecting the account email (the email used to sign in) in Share somewhere, "You" is excluded if the displayed name is long.

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/38435837/c3b82085-7861-4eee-906e-0143ce336fae

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~011a3106b6d163f6b3</li>
        <li>Upwork Job ID: 1777358786323771392</li>
        <li>Last Price Increase: 2024-04-08</li>
    </ul>
</details>

**Comment by m-natarajan** [comment]
@isabelastisser FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by isabelastisser** [comment]
@parasharrajat, please validate the issue and review the proposals above. Thanks!

**Comment by shawnborton** [comment]
I think we should remain as consistent as possible, so if we are showing this in the contacts list or LHN list, it should also be on this task confirmation page. 

cc @thienlnam in case you have any additional context as to why it was left out.

**Comment by shawnborton** [comment]
Ah great reminder. If we are going to do the subtitle everywhere, then maybe we can just drop the `(you)` from the contact name everywhere? Curious what others think about though.

**Comment by isabelastisser** [comment]
Thanks for the update, @parasharrajat!

**Comment by isabelastisser** [comment]
Not overdue. On hold.

**Comment by isabelastisser** [comment]
On hold.

**Comment by isabelastisser** [comment]
Hey @Expensify/applauseleads, can you please retest this? Thanks!

**Comment by kbecciv** [comment]
Checking

**Comment by kbecciv** [comment]
Issue is no longer reproducible, build 1.4.75.0

https://github.com/Expensify/App/assets/93399543/04933f9a-feb7-418c-98a5-4e55bcc5ff31

**Comment by isabelastisser** [comment]
Thanks for confirming, @kbecciv ! Closing.
